chef/site-cookbooks/wca/recipes/default.rb

```diff
@@ -173,10 +173,10 @@
 end
 
 # Run mailcatcher in every environment except production.
-if rails_env != "production"
+if node.chef_environment != "production"
```

Comment:
On staging, our `rails_env` is `"production"`, so we weren't actually installing and starting mailcatcher. This change causes mailcatcher to be installed and run on production.